### PR TITLE
Deprecate ANDROID_SDK_ROOT back to ANDROID_HOME 🔁

### PR DESCRIPTION
**Pull request by leotm**
Close: #3118

### CODE REVIEW

**Comment by leotm** [comment]
https://app.circleci.com/pipelines/github/facebook/react-native-website/5682

`Could not find a usable config.yml because you deleted the CircleCI OAuth app.` 🤔

**Review by leotm — COMMENTED** [review]
https://app.circleci.com/pipelines/github/facebook/react-native-website/5682

`Could not find a usable config.yml because you deleted the CircleCI OAuth app.` 🤔

File: docs/_getting-started-macos-android.md [hunk 1]
```diff
@@ -69,21 +69,30 @@ Next, select the "SDK Tools" tab and check the box next to "Show Package Details
 
 Finally, click "Apply" to download and install the Android SDK and related build tools.
 
-<h4>3. Configure the ANDROID_SDK_ROOT environment variable</h4>
+You can also run the following command after setting ANDROID_HOME.
+
+```shell
+sdkmanager "platforms;android-30" "system-images;android-30;default;x86_64" "system-images;android-30;google_apis;x86"
+sdkmanager "cmdline-tools;latest" "build-tools;30.0.2"
+```
```
Comment (cortinico) [inline docs/_getting-started-macos-android.md:77]: I would refrain from mentioning those commands as they require you to install the CLI tools which we're not asking the users to do IIRC on Mac

File: docs/_getting-started-macos-android.md [hunk 2]
```diff
@@ -69,21 +69,23 @@ Next, select the "SDK Tools" tab and check the box next to "Show Package Details
...
 Add the following lines to your `$HOME/.bash_profile` or `$HOME/.bashrc` (if you are using `zsh` then `~/.zprofile` or `~/.zshrc`) config file:
 
 ```shell
-export ANDROID_SDK_ROOT=$HOME/Library/Android/sdk
-export PATH=$PATH:$ANDROID_SDK_ROOT/emulator
-export PATH=$PATH:$ANDROID_SDK_ROOT/platform-tools
+export ANDROID_HOME=$HOME/Library/Android/sdk
+export PATH=$PATH:$ANDROID_HOME/emulator
+export PATH=$PATH:$ANDROID_HOME/tools
+export PATH=$PATH:$ANDROID_HOME/tools/bin
```
Comment (leotm) [inline docs/_getting-started-macos-android.md:82]: actually these couple better removed more minimal as before

don't have them in local `.zshrc` either w no troubles

Comment (leotm) [reply docs/_getting-started-macos-android.md]: unresolved: do linux as well

**Comment by cortinico** [comment]
> [app.circleci.com/pipelines/github/facebook/react-native-website/5682](https://app.circleci.com/pipelines/github/facebook/react-native-website/5682)
> `Could not find a usable config.yml because you deleted the CircleCI OAuth app.` 🤔

Thanks for flagging this. Let me get back to you

File: docs/_getting-started-macos-android.md (hunk 2 above)
Comment (leotm) [reply docs/_getting-started-macos-android.md]: resolved: https://github.com/facebook/react-native-website/pull/3266/commits/9cdb2e6247f35948332afbbea1033010892eb35e

**Comment by leotm** [comment]
> > [app.circleci.com/pipelines/github/facebook/react-native-website/5682](https://app.circleci.com/pipelines/github/facebook/react-native-website/5682)
> > `Could not find a usable config.yml because you deleted the CircleCI OAuth app.` 🤔
> 
> Thanks for flagging this. Let me get back to you

cool otherwise looks g2g after that

**Comment by leotm** [comment]
https://github.com/facebook/react-native-website/issues/3118#issuecomment-1316185555 prompted me to take another look

spotted and resolved a conflict https://github.com/facebook/react-native-website/pull/3266/commits/878779448ff5d2462a6d7daf9e7941c452fa1fac

prev CI issue seems to have fixed itself 🎉

**Review by cortinico — APPROVED** [review]

**Pull request merged**